<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DocSettingsPage</class>
 <widget class="QWidget" name="DocSettingsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="toolTip">
   <string>Add and remove compressed help files, .qch.</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Registered Documentation</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QListWidget" name="docsListWidget">
        <property name="selectionMode">
         <enum>QAbstractItemView::ContiguousSelection</enum>
        </property>
        <property name="uniformItemSizes">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="_4">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QToolButton" name="addButton">
          <property name="text">
           <string>Add...</string>
          </property>
          <property name="icon">
           <iconset resource="../coreplugin/core.qrc">
            <normaloff>:/core/images/plus-16.png</normaloff>:/core/images/plus-16.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="removeButton">
          <property name="text">
           <string>Remove</string>
          </property>
          <property name="icon">
           <iconset resource="../coreplugin/core.qrc">
            <normaloff>:/core/images/minus-16.png</normaloff>:/core/images/minus-16.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../coreplugin/core.qrc"/>
 </resources>
 <connections/>
</ui>
